pluto/build/edited.ini

```diff
@@ -17,136 +17,136 @@ url = https://fedoraplanet.org/edited
 
 
 [duffy]
-    name = Máirín Duffy 
+    name = Máirín Duffy
     link = http://mairin.wordpress.com/
     feed = http://mairin.wordpress.com/category/fedora/feed/
     avatar = https://duffy.fedorapeople.org/mo.png
     author = duffy
 
 [jsmith]
-    name = Jared Smith 
+    name = Jared Smith
     link = http://www.jaredsmith.net/topics/linux/fedora
     feed = http://www.jaredsmith.net/topics/linux/fedora/feed
     avatar = https://jsmith.fedorapeople.org/Images/Jared_Smith.png
     author = jsmith
 
 [hackergotchi]
-    name = Papadeas Pierros 
+    name = Papadeas Pierros
     link = http://pierros.papadeas.gr
     feed = http://pierros.papadeas.gr/?feed=rss2&tag=fedora
     avatar = https://fedoraproject.org/w/uploads/5/59/Hackergotchi-Ppapadeas.png
     author = hackergotchi
 
-; [mostafa21]
-;     name = Mostafa Daneshvar 
-;     ; link = link to the website
-;     feed = http://mostafadaneshvar.com/tag/fedora/feed/
-;     avatar = https://lashar.fedorapeople.org/mostafa21.png
-;     author = mostafa21
+[mostafa21]
+    name = Mostafa Daneshvar
+    ; link = link to the website
+    feed = http://mostafadaneshvar.com/tag/fedora/feed/
+    avatar = https://lashar.fedorapeople.org/mostafa21.png
+    author = mostafa21
 
-; [name]
-;     name = Gianluca Sforna 
-;     link = http://morefedora.blogspot.com
-;     feed = http://morefedora.blogspot.com/feeds/posts/default/-/justfedora/?alt=rss
-;     avatar = http://www.gravatar.com/avatar/a47795df6968fe5645cd3b6b94235e86?s=80
-;     ; author = fasname
```

Comment:
We should think about which fields are optional and which are required. This will have an impact on where conditionals are needed in the template files.